tests/integration/test_structsearch.py

```diff
@@ -0,0 +1,20 @@
+from pharmpy.internals.fs.cwd import chdir
+from pharmpy.tools import fit, run_structsearch
+
+
+def test_pkpd(tmp_path, load_model_for_test, testdata):
+    with chdir(tmp_path):
+        model = load_model_for_test(testdata / "nonmem" / "pheno_pd.mod")
+        start_res = fit(model)
+        res = run_structsearch(type='pkpd', route='iv', results=start_res, model=model)
+
+        assert len(res.summary_models) > 1
+        assert len(res.summary_tool) > 1
+        assert len(res.models) > 1
```

Comment:
In this case since the workflow is static and we always know how many models we expect, maybe it would be good to test the exact number of models?